app/experimenter/experiments/models/__init__.py

```diff
@@ -0,0 +1,41 @@
+from experimenter.experiments.models.legacy import (
+    Country,
+    default_all_platforms,
+    Experiment,
+    ExperimentBucketNamespace,
+    ExperimentBucketRange,
+    ExperimentChangeLog,
+    ExperimentChangeLogManager,
+    ExperimentComment,
+    ExperimentCommentManager,
+    ExperimentConstants,
+    ExperimentEmail,
+    ExperimentManager,
+    ExperimentVariant,
+    Locale,
+    Preference,
+    RolloutPreference,
+    VariantPreferences,
+)
+from experimenter.experiments.models.nimbus import NimbusExperiment
+
+__all__ = [
+    Country,
+    default_all_platforms,
+    Experiment,
+    ExperimentBucketNamespace,
+    ExperimentBucketRange,
+    ExperimentChangeLog,
+    ExperimentChangeLogManager,
+    ExperimentComment,
+    ExperimentCommentManager,
+    ExperimentConstants,
+    ExperimentEmail,
+    ExperimentManager,
+    ExperimentVariant,
+    Locale,
+    NimbusExperiment,
+    Preference,
+    RolloutPreference,
+    VariantPreferences,
+]
```

Comment:
Actually get rid of `Country` and `Locale` from this list, since those are defined here:

https://github.com/mozilla/experimenter/blob/main/app/experimenter/base/models.py

and this is what I was saying, accidental re-export 😆

Reply:
Oh I think I know why you put those here, becuase other modules are accidentally importing them from here instead of properly from `experimenter.base.models` okay if taking them out of here breaks tests then leave them in adn we can file a follow up to clean that up

Reply:
Yup, that's exactly why I had to import them.